easybuild/easyconfigs/o/openslide-python/openslide-python-1.1.1-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'openslide-python'
+version = '1.1.1'
+
+github_account = 'openslide'
```

Comment:
@lexming nit: please move to above `source_urls`?

Reply:
Fixed :+1: